bedrock/firefox/templates/firefox/new/desktop/firefox-new-refresh-all.html

```diff
@@ -0,0 +1,267 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% from "macros.html" import google_play_button, apple_app_store_button, fxa_email_form with context %}
+
+{% if switch('download_as_default') and not needs_data_consent(country_code) %}
+  {% set download_as_default_enabled = true %}
+{% else %}
+  {% set download_as_default_enabled = false %}
+{% endif %}
+
+{% macro default_browser_checkbox(id='default-browser-checkbox') -%}
+<label for="{{ id }}" class="default-browser-label hidden">
+  <input type="checkbox" id="{{ id }}" class="default-browser-checkbox">
+  Set Firefox as your default browser.
+</label>
+{%- endmacro %}
+
+{% extends "firefox/new/desktop/base.html" %}
+
+{% set referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=firefox-desktop' %}
+{% set ios_url = app_store_url('firefox', 'mozilla-org-firefox-new') %}
```

Comment:
i see source/campaign/content-like ct added to ios_url but empty for google play store button -- does it know how to pick up some? the play store uses utm_* that might be able to also be set?